### DISCUSSION TITLE
Kindly update request

### DISCUSSION THREAD

**Discussion by hkfsc** [kotatogram/kotatogram-desktop#363] [EN 1. General]
Hi, some functions in latest official TDesktop is really nice, like the new forward list design(I don't want to choose the icons any more), and when forward done, it can just open the chat with the one who we want to continue chat with, Live Streams, Folder adding in right click context, etc.

Currently beta version is based on TD 3.5.2, but they released 3.7 month ago, so the cored is already a little bit outdated.
I just want to kindly reminder that it's the time to have the core update, thanks!

**Comment by EricKotato** [comment]
See #119.
